src/filters.rs

```diff
@@ -232,6 +234,25 @@ pub fn last(input: &Value, _args: &[Value]) -> FilterResult {
     }
 }
 
+pub fn date(input: &Value, args: &[Value]) -> FilterResult {
+    if args.len() != 1 {
+        return Err(FilterError::InvalidArgumentCount(format!("expected 1, {} given", args.len())));
+    }
+
+    let date = match input {
+        &Value::Str(ref s) => try!(DateTime::parse_from_str(&s, "%d %B %Y %H:%M:%S %z")
```

Comment:
So the original liquid has a `to_date` function that I think is a good idea to implement for us as well https://github.com/Shopify/liquid/blob/master/lib/liquid/utils.rb#L63

However, we don't have to do it in this PR. I'll make a follow-up issue.